packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx

```diff
@@ -92,8 +92,12 @@ export function FetchingWrappedAssetCard(props: FetchingWrappedAssetCardProps) {
       renderDragHandle: props.renderDragHandle,
       onEdit,
       onRemove,
-      useLocalizedEntityStatus: props.sdk.parameters.instance.useLocalizedEntityStatus,
-      isLocalized: !!('localized' in props.sdk.field && props.sdk.field.localized), // missing in types :(
+      useLocalizedEntityStatus:
+        props.sdk.parameters.instance.useLocalizedEntityStatus &&
+        props.sdk.parameters.instance.isLocalePublishingEnabled,
```

Comment:
`useLocalizedEntityStatus` should be the same as `isLocalePublishingEnabled`?

useLocalizedEntityStatus resolves to the feature flag / entitlement check

Reply:
Where is that being set?

Reply:
web app: src/javascripts/app/entity_editor/EntityField/useWidgetApi.ts#L226